en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+<td>
+
+API Id
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>topSlowestAPIs</strong></td>
+<td valign="top">[<a href="#slowapi">SlowAPI</a>]</td>
+<td>
+
+Returns a list of top slowest APIs by the response latency (95th percentile value) within the specified time range
+for the given combination of tenant ID, environment ID, and organization ID.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">filter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">limit</td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td>
+
+Limit for the slow APIs list
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getLatency</strong></td>
+<td valign="top"><a href="#apilatency">APILatency</a></td>
+<td>
+
+Returns different latency category values (95th percentile values) for a given API over time for each time
+interval (calculates according to the granularity value) within the specified time range for the given combination
+of tenant ID, environment ID, and organization ID.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">latencyFilter</td>
+<td valign="top"><a href="#latencyfilter">LatencyFilter</a>!</td>
+<td>
+
+Latency filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getOverallLatencyByAPI</strong></td>
+<td valign="top"><a href="#overalllatency">OverallLatency</a></td>
+<td>
+
+Returns the overall response latency (95th percentile value) for an API across all the data available time range for
+the given combination of tenant ID, environment ID, and organization ID. Optionally, you can configure the `filter`
+parameter to further filter the results by a time range.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">filter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a></td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">apiId</td>
+<td valign="top"><a href="#id">ID</a>!</td>
+<td>
+
+API Id
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getCacheHitsAndMisses</strong></td>
+<td valign="top"><a href="#cachehits">CacheHits</a></td>
+<td>
+
+Return the total number of response cache hits, misses and hit percentage for an API over time for each time
```

Comment:
'over time for each time interval (calculates according to the granularity value) within the specified time range" is too confusing. Consider replacing this phrase with the following:

"for each time granularity within the specified time range"